lib/mutant/mutator/node/send.rb

```diff
@@ -27,7 +27,8 @@ class Send < self
           to_a:          %i[to_ary],
           to_h:          %i[to_hash],
           at:            %i[fetch],
```

Comment:
Doesn't seem out of scope. I agree I think if we're mutating `:[]` and `:fetch` that means `:at` should be mutated as well